pep/app/pods/components/page/sidebar/component.ts

```diff
@@ -1,17 +1,58 @@
 import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
 import { inject as service } from '@ember/service';
 import { action } from '@ember/object';
-import MediaService from 'ember-responsive/services/media';
+import { next } from '@ember/runloop';
+import { htmlSafe } from '@ember/template';
 
 import SidebarService from 'pep/services/sidebar';
+import FastbootMediaService from 'pep/services/fastboot-media';
+import { SIDEBAR_HANDLE_WIDTH, SIDEBAR_WIDTH } from 'pep/constants/dimensions';
 
 interface PageSidebarArgs {
     side: 'left' | 'right';
+    resizable?: boolean;
```

Comment:
From what I can see, these are not actually being passed in? If that's the case, why are they in the `PageSidebarArgs`?

Reply:
they are optional right now and are used in the getters below, but just added some basic knobs here in case resize-ablity needs to be customizable later.